app/views/entities/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+  <%= form_with model: @entity, method:'post', 
+      class:'form',
+      url: entities_path(category_id: @category.id) do |form| %>
+  <div class="flex main-bg-color color-white justify-center align-center justify-between height-40px">
+    <%= link_to "🡄", entities_path(category_id: @category.id), method: :get, class:"back ml-10" %>
+    <h2>Add Transaction</h2>
+    <div class="actions color-white mr-10">
+      <%= form.submit "Create"%>
+    </div>
+  </div>
+    <div class="field">
+      <%= form.text_field :name, autofocus: true, placeholder: "Name" %>
+      <%= form.number_field :amount, autofocus: true, placeholder: "Amount" %>
+    </div>
+  <% end %>
```

Comment:
Hello @Gambit142 , thank you for the review.
To create a transaction, the user necessarily needs to select the category on the home page. I didn't implement this feature on purpose because it would be kind of silly. If the user is already in the category to create the transaction, he already has the category selected. If the reviewer wants, I can implement a select input for this in the form, but I'll remove it when I publish the demo.

Reply:
@franciscoPonceDev , kindly read the project requirements a second time. Remember that the relationship between the group and entity is a many-to-many relationship. I will only review based on the project requirements. That is the reason I insisted that you have a form input that maps to the category when creating a new transaction. I guess working on a project based on requirements in a professional world is not such a _**silly idea**_. 😄

Reply:
@Gambit142 Thanks. As I said, if the reviewer wants it, I will implement the feature. But it's a silly feature. Using critical thinking to point out improvements and/or unnecessary things (like this silly feature) is also a developer duty. Having said that, I will implement the feature anyway.